packages/www/pages/docs/guides/debugging-guide.mdx

```diff
@@ -0,0 +1,71 @@
+import DocsLayout from "../../../components/DocsLayout";
+
+<DocsLayout>
+
+
+# Debugging Livepeer.com Stream Playback
+
+This step-by-step guide will help you debug common stream playback problems. Try these steps before reaching out to Livepeer.com directly.
+
+We will use the [Livepeer.com test player](/app/test-player) to debug. You must be logged into the Livepeer.com dashboard. 
+
+#### Prerequisite: Locate your playback URL. 
+
+Your playback URL is specific to the stream ingest region. 
+- This [Livepeer.com API guide](/docs/guides/api/playback-a-live-stream) touches on finding your playback URL by combining a base url with a `playbackId`. If you don't know your stream `playbackId`, you can retrieve it via [`GET /stream/<id>`](/docs/api/stream/get-stream).
+- You can also find the playback URL on the Livepeer.com dashboard stream page, `https://livepeer.com/app/stream/<stream-id>`. Learn more [here](/docs/guides/dashboard/playback-a-stream).
+
+## Step 1
+
+Load your playback URL in the [Livepeer.com test player](/app/test-player). If you get a "Stream not found" error, confirm that your URl is correct. It should look something like this:  https://region-cdn.livepeer.com/hls/123456abcdef7890/index.m3u8
+
+## Step 2
+
+Does anything playback in *your* own player?
+
+- **No:** Go to the next step.
+- **Yes, but the quality is poor:** Go to step 5.
```

Comment:
Could we add a third choice? Wording up to you, this is just an example: "**Yes, it works and quality is great** Check ____ and ____. There may be something wrong with your player configuration. For more information on what you could do to debug the issue, take a look at ____ and ____. Common mistakes are ____ and _____. ". 

I know you have it in the next step, but would be useful here as well, since it is possible that stream plays well on their player. Would be great to include resources they could access that could be helpful.